docs/configuration/mongodb-event-flattening.asciidoc

```diff
@@ -16,6 +16,11 @@ Please see below for a descriptions of known limitations of this transformation.
 This SMT is supported only for the MongoDB connector.
 ====
 
+[CAUTION]
+====
+There is a https://issues.jboss.org/browse/DBZ-624[known issue] for update messages when MongoDB 3.6 is used.
```

Comment:
I think we should be a bit more specific and state that the SMT won't work with 3.6 for the time being.